tools/icu/shrink-icu-src.py

```diff
@@ -128,7 +128,7 @@ def icu_info(icu_full_path):
 readme_name = os.path.join(options.icudst, "README-FULL-ICU.txt" )
 
 # Now, print a short notice
-fi = open(readme_name, 'wb')
+fi = open(readme_name, 'w')
```

Comment:
It removes boilerplate if we format our message as a triple quoted string and then print it to file in a single print().
Replace all remaining lines with:
```suggestion
msg_fmt = """\
ICU sources - auto generated by shrink-icu-src.py\n
This directory contains the ICU subset used by --with-intl=full-icu
It is a strict subset of ICU {} source files with the following exception(s):
* {} : compressed data file\n\n
To rebuild this directory, see ../../tools/icu/README.md\n"""

with open(readme_name, 'w') as out_file:
    print(msg_fmt.format(icu_ver_major, dst_cmp_datafile), file=out_file)
```